src/app/components/home/home.component.spec.ts

```diff
@@ -39,7 +46,7 @@ describe('HomeComponent', () => {
   it('should render button to create the room', async(() => {
     fixture.detectChanges();
     const compiled = fixture.debugElement.nativeElement;
-    expect(compiled.querySelector('button').textContent).toContain('Create room');
+    expect(compiled.querySelector('#create').textContent).toContain('Create room');
```

Comment:
this is HTML's ID, right? I'd suggest switching to classes or data-attributes with things like `create`, because it is such a common word that there is a pretty big chance that it can repeat itself. It's gonna work ofc, but im just thinking about being semantically correct ;)

Reply:
You are right, this was "hotfix" for test, I will improve name